### PR TITLE
Added address output to styles

### PR DESCRIPTION
**Pull request by JNATION85**
We now have the what an address output looks like in the 'Styles' tab.

Need to consider renaming the 'Address' component to 'Address Input' to match the 'Address Output' component

### CODE REVIEW

**Comment by rmccar** [comment]
I don't think we can rename the address pattern to "Address Input" because it wouldn't fit in with the "Ask users for..." section heading
image

File: src/styles/typography/index.njk [hunk 1]
```diff
@@ -79,12 +79,19 @@ The base font size for paragraphs is 18px.
...
 
 ## Text indent
-Use when providing an address or contact information
+Use when providing contact information.
 
 {{
     patternlibExample({"path": "styles/typography/text-indent/examples/text-indent/index.njk", "componentFolderPath": "components/text-indent"})
 }}
 
+## Addresses
```
Comment (rmccar) [inline src/styles/typography/index.njk:88]: If we do want to differentiate this from the Address pattern we could change this line to "Address output" which would also update the nav link

Comment (jrbarnes9) [reply src/styles/typography/index.njk:88]: I think we just need to change the folder name of the current component from `addresses` to `address-input`.  The frontmatter can stay as it is.

File: src/components/address-output/_macro.njk [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{% macro onsAddressOutput(params) %}
+    <div class="address-output{% if params.classes is defined and params.classes %} {{ params.classes }}{% endif %}">
+        <p class="address-output__lines">
+            {% if params.line1 is defined and params.line1 %}
+                {{params.line1}}<br>
```
Comment (jrbarnes9) [inline src/components/address-output/_macro.njk:5]:
```suggestion
                {{ params.line1 }}<br>
```

File: src/components/address-output/_macro.njk [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+            {% endif %}
+            {% if params.line2 is defined and params.line2 %}
+                {{params.line2}}<br>
```
Comment (jrbarnes9) [inline src/components/address-output/_macro.njk:8]:
```suggestion
                {{ params.line2 }}<br>
```

File: src/components/address-output/_macro.njk [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+            {% endif %}
+            {% if params.town is defined and params.town %}
+                {{params.town}}<br>
```
Comment (jrbarnes9) [inline src/components/address-output/_macro.njk:11]:
```suggestion
                {{ params.town }}<br>
```

File: src/components/address-output/_macro.njk [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+            {% endif %}
+            {% if params.postcode is defined and params.postcode %}
+                {{params.postcode}}<br>
```
Comment (jrbarnes9) [inline src/components/address-output/_macro.njk:14]:
```suggestion
                {{ params.postcode }}
```

File: src/styles/typography/address-output/examples/address-output/index.njk [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{% from "components/address-output/_macro.njk" import onsAddressOutput %}
+{{-
+    onsAddressOutput({
+        line1: 'Abingdon Road',
+        line2: 'Goathill',
+        town: 'Dorset',
```
Comment (jrbarnes9) [inline src/styles/typography/address-output/examples/address-output/index.njk:6]: Dorset's not a town 😜

File: src/components/address-output/_macro.njk [hunk 7]
```diff
@@ -0,0 +1,18 @@
+{% macro onsAddressOutput(params) %}
+    <div class="address-output{% if params.classes is defined and params.classes %} {{ params.classes }}{% endif %}">
+        <p class="address-output__lines">
+            {% if params.line1 is defined and params.line1 %}
```
Comment (jrbarnes9) [inline src/components/address-output/_macro.njk:4]: I think we could do with adding 'unit' and 'organisation' address lines to cater for CE and business addresses

File: src/styles/typography/address-output/examples/address-output/index.njk (hunk 6 above)
Comment (JNATION85) [reply src/styles/typography/address-output/examples/address-output/index.njk:6]: It is now

File: src/styles/typography/index.njk (hunk 1 above)
Comment (rmccar) [reply src/styles/typography/index.njk:88]: Fair enough, may as well also do that in this PR then

**Comment by rmccar** [comment]
[PR #1514 - Postal address style](https://trello.com/c/UI8PDAJN/542-pr-1514-postal-address-style)

**Review by boxadesign — DISMISSED** [review]

**Review by boxadesign — DISMISSED** [review]

File: src/styles/typography/index.njk [hunk 8]
```diff
@@ -119,7 +126,7 @@ Each list item should be a complete sentence, therefore start with an uppercase
 
 You do not need to introduce a numbered list with a lead-in line.
 
-To use a numbered/orderd list when using the list macro you need to not be using a bare list and set the `element` parameter to "ol".
+To use a numbered or ordered list when using the list macro you need to not be using a bare list and set the `element` parameter to "ol".
```
Comment (jrbarnes9) [inline src/styles/typography/index.njk:129]:
```suggestion
To use a numbered list when using the list macro you need to not be using a bare list and set the `element` parameter to "ol".
```

**Review by rmccar — APPROVED** [review]

**Review by jrbarnes9 — APPROVED** [review]

**Review by jrbarnes9 — APPROVED** [review]

**Review by rmccar — APPROVED** [review]

**Pull request merged**